sdk/confidentialledger/azure-confidentialledger/CHANGELOG.md

```diff
@@ -1,14 +1,18 @@
 # Release History
 
-## 1.0.1 (Unreleased)
+## 1.1.0 (2022-12-21)
+Add `azure.confidentialledger.receiptverification` module for Azure Confidential Ledger write transaction receipt verification.
```

Comment:
Was it discussed in archboard that this live in a submodule?
I can't remember - I will go back and listen to the recording...
If so - this name seems very long...

Reply:
Indeed, during the SDK review, we just decided to move this library inside the existing Data Plane library. We initially had in mind to move everything inside `_patch.py` but we realized that it's not ideal. It would make the code messier and more difficult to maintain, and it would be harder for customers to audit the receipt verification code, which is one of the reasons we're adding it to the SDK. Additionally, the new verification functionality would only be visible under `azure.confidentialledger`, while I believe it would be better placed in a different sub-module, as it is a slightly different utility from the other operations allowed by the SDK.

Is having a different sub-module problematic for this type of library? Besides, the name can be different: we can do `azure.confidentialledger.receipt` for example or any other better / shorter naming we can come up with. For reference, we already have another sub-module in this library called `azure.confidentialledger.certificate`.

Reply:
@johanste @lynshi Do you have any opinion about this? I think having this library as a sub-module should be fine and I don't see any specific problems with it. In terms of library naming, we are flexible and we can do a shorter naming (`azure.confidentialledger.receipt`) if preferred.

Reply:
Renamed to `azure.confidentialledger.receipt`, let me know if that's ok